packages/app-bridge/src/types/Color.ts

```diff
@@ -134,4 +143,32 @@ export type ColorCreate = {
     alpha: number;
 };
 
-export type ColorPatch = Partial<Omit<Color, 'id' | 'hex' | 'hue' | 'saturation' | 'lightness'>>;
+export type ColorPatch = {
```

Comment:
Is there an advantage in defining all the color patch properties in a single object of top level properties. For example, I see how properties `c`, `m`, `y`, `k` could all be encapsulated into a `cmyk` object that would hold all cmyk related color properties (including coated, uncoated and newspaper). Same goes for pantone properties. What do you think?

Reply:
ColorPatch should basically stay the same as before